accounts/forms.py

```diff
@@ -0,0 +1,10 @@
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+
+CustomUser = get_user_model()
+
+class SignupForm(UserCreationForm):
+    
+    class Meta(UserCreationForm.Meta):
+        model = CustomUser
+        fields = ('username', 'email', 'password1', 'password2', 'age')      

```

Comment:
```suggestion
     class Meta:
        model = CustomUser
        fields = ('username', 'email', 'age')      
```
でも大丈夫です。変更するかしないかは任せます